Bottom: 00d8e7ae47314f45e85d18dd47446d14ac08854b
Top:    dd51ed548b733bdea739d35f27e779efcb4683c7
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-29 09:09:08 -0500

Refresh of codebase-generator

---
diff --git a/roles/buildbot-master/files/master.cfg b/roles/buildbot-master/files/master.cfg
index c14bd7dd..5a366551 100644
--- a/roles/buildbot-master/files/master.cfg
+++ b/roles/buildbot-master/files/master.cfg
@@ -172,7 +172,7 @@ def codebaseGenerator(chdict):
     """
 
     giturl = giturlparse(chdict['repository'])
-    return giturl.owner
+    return '/'.join((giturl.owner, giturl.repo))
 
     # path = urlparse(chdict['revlink']).path.lstrip('/')
